ParselyTracker/ParselyTracker.swift

```diff
@@ -1,8 +1,7 @@
 import Foundation
+import UIKit
```

Comment:
Do we need `UIKit` imported here? It doesn't appear to be used anywhere.

Reply:
We do—`UIApplication` and `UIBackgroundTaskIdentifier` come from that module. `addApplicationObservers()` started emitting compiler errors after I removed `import Reachability`.